cmd/docker-notation/docker/manifest.go

```diff
@@ -0,0 +1,124 @@
+package docker
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/distribution/distribution/v3/manifest/schema2"
+	"github.com/notaryproject/notation/pkg/config"
+	"github.com/notaryproject/notation/pkg/docker"
+	"github.com/opencontainers/go-digest"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// GenerateManifest generate manifest from docker save
+func GenerateManifest(reference string) ([]byte, error) {
```

Comment:
Is it the same as the one in `docker-generate/manifest`?

Reply:
This one is a subset of the one in `docker-generate/manifest`.